### ISSUE TITLE
Create single binary version for Windows

### ISSUE THREAD

**Issue by voltagex** [dschep/ntfy#79]
http://www.pyinstaller.org/
https://www.appveyor.com/docs/deployment/github/

I'm finding pyinstaller quite difficult to use, but I'll have a branch up shortly.

**Comment by voltagex** [comment]
I realise an Issue isn't really the place for documentation, but it can be moved later.

I used https://github.com/ntoll/mu/blob/master/package/pyinstaller.spec as a reference

```powershell
#in a newly created env, with a clone of ntfy from my pyinstaller branch
pip install git+git://github.com/pyinstaller/pyinstaller.git@develop
pip install -r requirements.txt
pyinstaller --onedir .\ntfy.spec
dist\ntfy\ntfy.exe
```

This still requires some dlls to be shipped with ntfy.exe, unfortunately.

**Comment by voltagex** [comment]
This is not something I can accomplish at the moment - I had a lot of trouble with PyInstaller. Closes #80

**Comment by dschep** [comment]
No worries, I'd ended up giving up on it too when I tried :sad:
